modules/elements-color-picker/assets/js/editor/commands/end.js

```diff
@@ -0,0 +1,26 @@
+import CommandBase from 'elementor-api/modules/command-base';
+
+export class End extends CommandBase {
```

Comment:
```suggestion
export class Apply extends CommandBase {
```

Reply:
Maybe we should also change the `start` command to something like `pick`?